libs/utils/utils.js

```diff
@@ -453,6 +453,8 @@ async function loadPostLCP(config) {
 }
 
 export async function loadDeferred(area) {
+  const links = area.querySelectorAll('a:is([href*="#_blank"])');
+  [...links].map((link) => link.setAttribute('target', '_blank'))
```

Comment:
This should be in the `decorateLinks` function. There's no need to hit the DOM again looking for links.

Reply:
Makes sense - thx!